src/stock-on-hand-mobile/stock-on-hand.wrapper.jsx

```diff
@@ -42,8 +45,14 @@ import MetaTags from 'react-meta-tags';
                             <meta name='viewport' content='width=device-width, initial-scale=1.0'/>
                         </MetaTags>
                         <StockOnHandApp
+                            $q={$q}
```

Comment:
What does mean this '$q' service? I know that we cannot change the name defined in angular but it could be good to (as I mentioned above) rename this prop into the name that could precise the meaning of this service. For example:
```
                       <StockOnHandApp
                            {betterNameForThisQService}={$q}
                            facilityFactory={facilityFactory}
                            offlineService={offlineService}
                            facilityService={facilityService}
                            programService={programService}
                            authorizationService={authorizationService}
                            currentUserService={currentUserService}
                            permissionService={permissionService}
                        />
```